client/src/pages/layout/Navbar.js

```diff
@@ -55,46 +54,56 @@ const Navbar = () => {
-                                component={NavLink}
-                                to="/chat"
-                            >
-                                <ListItem>
-                                    <Typography variant="h6">Chats</Typography>
-                                    <Badge badgeContent={12} className={classes.badge} />
-                                </ListItem>
-                            </Link>
+                            <ListItem>
+                                <NavLink
+                                    color="inherit"
+                                    variant="inherit"
+                                    underline="none"
+                                    to="/forum"
+                                    style={{ opacity: '50%' }}
+                                    activeStyle={{
+                                        opacity: '100%',
+                                    }}
+                                    className={classes.linkStyles}
+                                >
+                                    Forum
+                                </NavLink>
+                            </ListItem>
+                            <ListItem>
+                                <NavLink
+                                    color="inherit"
+                                    variant="inherit"
+                                    underline="none"
+                                    to="/groups"
+                                    activeStyle={{
+                                        opacity: '100%',
+                                    }}
+                                    className={classes.linkStyles}
+                                >
+                                    Groups
+                                </NavLink>
+                            </ListItem>
+                            <ListItem>
+                                <NavLink
+                                    color="inherit"
+                                    variant="inherit"
+                                    underline="none"
+                                    to="/chat"
+                                    activeStyle={{
+                                        opacity: '100%',
+                                    }}
+                                    className={classes.linkStyles}
+                                >
+                                    Chats
+                                </NavLink>
+                                <Badge badgeContent={12} className={classes.badge} />
+                            </ListItem>
                         </List>
                     </div>
                     <div className={classes.profile}>
                         <Avatar
-                            alt="profiel_img"
-                            src={profile.imageUrl.length ? profile.imageUrl : profileImg}
+                            alt="profile_img"
+                            src={profile.imageUrl}
```

Comment:
If we're using the default icon we in DropzonePicture we want to use it here as well